package tv.anilibria.module.data.network.entity.app.auth

class SocialAuthException : RuntimeException()